modules/consensus/synchronize.go

```diff
@@ -31,13 +31,29 @@ var (
 		Testing:  types.BlockHeight(3),
 	}).(types.BlockHeight)
 
-	// sendBlocksTimeout is the timeout for the SendBlocks RPC.
+	// sendBlocksTimeout is the timeout for the SendBlocks RPC, and for the
+	// depricated RelayBlock RPC.
 	sendBlocksTimeout = build.Select(build.Var{
 		Standard: 5 * time.Minute,
 		Dev:      40 * time.Second,
 		Testing:  5 * time.Second,
 	}).(time.Duration)
 
+	// sendBlkTimeout is the timeout for the SendBlocks RPC, and for the
+	// depricated RelayBlock RPC.
+	sendBlkTimeout = build.Select(build.Var{
+		Standard: 4 * time.Minute,
+		Dev:      30 * time.Second,
+		Testing:  4 * time.Second,
+	}).(time.Duration)
+
+	// relayHeaderTimeout is the timeout for the RelayHeader RPC.
+	relayHeaderTimeout = build.Select(build.Var{
+		Standard: 3 * time.Minute,
```

Comment:
It is rather generous, but the problem is not the data volume it's the round trip time. Especially nodes in China need a lot of extra time to get their packets around.